app/boards/shields/iris/Kconfig.defconfig

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Please add a copyright header here. (I know we're missing some in `main`, trying to avoid even more getting in without them. Thanks.